README.md

```diff
@@ -314,9 +318,9 @@ With:
 </IfModule>
 ```
 
-#### Command Injection won't work.
+#### Command Injection won't work
 
--A. Apache may not have high enough privileges to run commands on the web server. If you are running DVWA under Linux make sure you are logged in as root. Under Windows log in as Administrator.
```

Comment:
@digininja  Just for a double check, is this `-A. ` a typo or intended?

Reply:
Typo. I think it used to be question and answer so this was the answer to what is now the title.

Reply:
I see. Thanks for the confirmation!